lib/src/nodes/config_node_include.cc

```diff
@@ -0,0 +1,40 @@
+#include <internal/nodes/config_node_include.hpp>
+#include <internal/nodes/config_node_simple_value.hpp>
+
+using namespace std;
+
+namespace hocon {
+
+    config_node_include::config_node_include(vector<unique_ptr<abstract_config_node>> children,
+                                             config_include_kind kind) :
+        _children(move(children)), _kind(kind) { }
+
+    token_list config_node_include::get_tokens() const {
+        token_list tokens;
+        for (auto&& node : _children) {
+            for (auto&& token : node->get_tokens()) {
```

Comment:
Instead of looping over get_tokens, I think you could do `tokens.insert(tokens.end(), node->get_tokens().begin(), node->get_tokens().end())`. I wish that weren't longer, because conceptually it's simpler and performs better.